config.toml

```diff
@@ -5,23 +5,68 @@ theme = "learn"
   # Change default color scheme with a variant one. Can be "red", "blue", "green".
   themeVariant = "blue"
 
+  # Un titre est défini par défaut lorsque vous utilisez un raccourci dans le menu. Utilisez ce paramètre pour le cacher.
+  disableShortcutsTitle = true
+
+  # When using mulitlingual website, disable the switch language button.
+  disableLanguageSwitchingButton = false
+
+  # Hide Next and Previous page buttons normally displayed full height beside content
+  disableNextPrev = true
+
+  # Hide breadcrumbs in the header and only show the current page title
+  disableBreadcrumb = true
+
+  # Disable search function. It will hide search bar
+  disableSearch = false
+
+  # Description of the site, will be used in meta information
+  description = ""
```

Comment:
Instead of leaving this blank, let's put something like "landing page for Rotational Labs educational content on globally distributed systems"

Reply:
Got it !